src/pyflakes/mod.rs

```diff
@@ -2248,13 +2244,14 @@ mod tests {
 
     #[test]
     fn unused_package_with_submodule_import() -> Result<()> {
+        // TODO Is that expected?
         // When a package and its submodule are imported, only report once.
```

Comment:
Unsure about this test. Why not report twice?

Reply:
Yeah reporting twice is better, I think.

Reply:
(These are literally the Pyflakes tests ported to Rust, so if we change behavior (as we're doing in this PR), we'll likely need to tweak the expectations of the tests.)